### ISSUE TITLE
search interface 

### ISSUE THREAD

**Issue by pirco** [pirco/social-society-dawn#10]
see figma design. and check https://flatvernacular.com/ for possible implementation. 
1. on desktop, the search icon expands into a search field (make sure it fits on small dsktop)
2. on mobile, the search field appears at top of menu

**Comment by pirco** [comment]
like so:
<img width="573" alt="Screenshot 2024-06-19 at 00 26 25" src="https://github.com/pirco/social-society-dawn/assets/146350/fd9e1fcc-9a0f-4668-89ea-53a212f60fd0">

**Comment by lastant** [comment]
Implemented on desktop

**Comment by pirco** [comment]
ok great. I did make a few adjustments, to make the expanded search field fit on smaller desktops and to make sure it doesn't change height of the menu bar. NOW:

1. the search results dropdown/modal shifts off to the right of the canvas once you start searching. can you right-align the whole search results dropdown?

2. can you change the font in the search results dropdown to  var(--font-body-family); ?

3 next step: please implement a search form on mobile menu. once again, you can lean on flatvernacular to see what it should look like. 

<img width="520" alt="Screenshot 2024-06-20 at 01 59 42" src="https://github.com/pirco/social-society-dawn/assets/146350/41a02049-5e5e-4fba-9971-ada89155b97e">

basically, search field shown by default. 

thanks

**Comment by lastant** [comment]
Added mobile
